### PR TITLE
Add jenkins script.

### PR DESCRIPTION
**Pull request by mklauber**

### CODE REVIEW

File: jenkins.sh [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+    PATH=$WORKSPACE/venv/bin:$PATH
+    if [ ! -f "venv/bin/activate" ]; then
+            virtualenv venv
+    fi
+    . venv/bin/activate
+
+    # Update the virtualenv with the latest requirements.
+    pip install setuptools==24.0.3
+    pip install -r MyJobs/requirements.txt --download-cache=/tmp/$JOB_NAME
+    pip install MySQL-python
```
Comment (tdphillips) [inline jenkins.sh]: MySQL-python 1.2.5 is in requirements.txt already. Do we need a more up to date version?

Comment (mklauber) [inline jenkins.sh]: No.  That line can/should be removed.

**Comment by aljordan82** [comment]
The /static directory also needs to be watched.

**Comment by tdphillips** [comment]
Do we have any tests for files residing in the /static directory?

**Comment by aljordan82** [comment]
@tdphillips It looks like the js files in /static are not tested at all. I thought they were tested.

**Pull request merged**